download.phtml

```diff
@@ -41,17 +41,17 @@
   <h3><?php echo T_("Sandbox")?></h3>
   <ul class="imagelist">
     <li class="imagelistitem" style="background-image: url(<?=$top_dir?>/images/icons/flatpak.png);">
-      <p class="justify"><?php printf (T_("Flatpaks are bundles, which contain beneath the program also all required libraries. If your distribution is too old, to offer all required libraries, but has the package <a href=\"%s\">Flatpak</a>, you can install a recent version of <span class=\"gnucash\">GnuCash</span> from"), $URL_FLATPAK)?></p>
+      <p class="justify"><?php printf (T_("Flatpaks are programs that bundle in all required libraries. If you have <a href=\"%s\">Flatpak</a> installed, you can grab any recent version of <span class=\"gnucash\">GnuCash</span> from"), $URL_FLATPAK)?></p>
```

Comment:
According to Wikipedia, "Flatpak is a utility for software deployment and package management for Linux." 

Searches online do not show any usage where the resulting bundle is referred to as a "flatpak".

Suggest changing this section to clearly fall under the Linux heading as an alternative to the various software center installation methods.  Perhaps something like:

An alternative to using your distribution's software manager is to use the Flatpak management system, which bundles all the necessary libraries for you. This can ease the installation of newer versions of GnuCash on older Linux machines. 

Also, "grab" is a decidedly informal term with many negative associations. Suggest using more appropriate terminology, such as "download."